soccer/Configuration.cpp

```diff
@@ -2,35 +2,35 @@
 
 #include <QTreeWidget>
 #include <QTreeWidgetItem>
-
-#include <stdio.h>
-#include <assert.h>
+#include <cassert>
+#include <cstdio>
+#include <iostream>
+#include <utility>
 
 std::list<Configurable*>* Configurable::_configurables;
 
 // Role for tree column zero for storing ConfigItem pointers.
 static const int ConfigItemRole = Qt::UserRole;
 
-Q_DECLARE_METATYPE(ConfigItem*)  // FIXME: verify this
+Q_DECLARE_METATYPE(ConfigItem*)  // FIXME: verify this NOLINT
```

Comment:
Oh god I don't trust anything in the config system. It's slated for removal as soon as we have ROS.